switch-configuration/config/switchtypes

```diff
@@ -13,57 +13,57 @@
-LosAngelesAB	7	503	2001:470:f325:503::200:7	hiRoom			D.9		Normal
-LosAngelesC	8	503	2001:470:f325:503::200:8	hiRoom			D.9		Quiet
-CenturyAB	9	503	2001:470:f325:503::200:9	hiRoom			D.9		Normal
-SanLorenzoB	10	503	2001:470:f325:503::200:10	hiIDF			D.1		Loud
-CenturyCD	11	503	2001:470:f325:503::200:11	hiRoom			D.9		Normal
-SantaMonicaA	12	503	2001:470:f325:503::200:12	hiRoom			D.9		Normal
-CatalinaB	13	503	2001:470:f325:503::200:13	hiRoom			E.9		Normal
-AVSwitch	14	503	2001:470:f325:503::200:14	hiAV			?.9		Normal
-SpareD		15	503	2001:470:f325:503::200:15	hiIDF			Z.9		??
-SpareE		16	503	2001:470:f325:503::200:16	hiIDF			Z.9		??
-CatalinaC	17	503	2001:470:f325:503::200:17	hiRoom			E.9		Normal
-ExpoA1		18	503	2001:470:f325:503::200:18	Booth			L.2		Loud
-CatalinaD	19	503	2001:470:f325:503::200:19	hiRoom			E.9		Normal
-NewportA	20	503	2001:470:f325:503::200:20	hiRoom			D.9		Quiet
-NOC		21	503	2001:470:f325:503::200:21	NOC			?.2		Normal
-ExpoB1		22	503	2001:470:f325:503::200:22	Booth			L.2		Loud
-CTF1		23	503	2001:470:f325:503::200:23	hiCTF			Z.9		Normal
-ExpoC1		24	503	2001:470:f325:503::200:24	Booth			L.2		Loud
-NewportB	25	503	2001:470:f325:503::200:25	hiRoom			D.9		Quiet
-CTF2		26	503	2001:470:f325:503::200:26	hiCTF			Z.9		Normal
-SanLorenzoC	27	503	2001:470:f325:503::200:27	hiRoom			D.9		Normal
-ExpoA2		28	503	2001:470:f325:503::200:28	Booth			L.2		Loud
-ExpoB2		29	503	2001:470:f325:503::200:29	Booth			L.2		Loud
-SanLorenzoD	30	503	2001:470:f325:503::200:30	hiRoom			D.9		Normal
-NewportC	31	503	2001:470:f325:503::200:31	hiRoom			E.9		Quiet
-RegDesk		32	503	2001:470:f325:503::200:32	registration		?.9		Normal
-ExpoC2		33	503	2001:470:f325:503::200:33	Booth			L.2		Normal
-ExpoA3		34	503	2001:470:f325:503::200:34	Booth			L.2		Loud
-ExpoB3		35	503	2001:470:f325:503::200:35	Booth			L.2		Loud
-ExpoC3		36	503	2001:470:f325:503::200:36	Booth			L.2		Normal
-SantaMonicaB	37	503	2001:470:f325:503::200:37	hiRoom			D.9		Quiet
-ExpoA4		38	503	2001:470:f325:503::200:38	Booth			L.2		Normal
-ExpoB4		39	503	2001:470:f325:503::200:39	Booth			L.2		Normal
-SantaMonicaC	40	503	2001:470:f325:503::200:40	hiRoom			D.9		Quiet
-ExpoC4		41	503	2001:470:f325:503::200:41	hiRoom			L.2		Normal
-ExpoA5		42	503	2001:470:f325:503::200:42	hiRoom			L.2		Normal
-ExpoB5		43	503	2001:470:f325:503::200:43	hiRoom			L.2		Normal
-BelAir		44	503	2001:470:f325:503::200:44	hiRoom			E.9		Loud
-SanLorenzoA	45	503	2001:470:f325:503::200:45	hiRoom			D.9		Normal
-SanLorenzoE	46	503	2001:470:f325:503::200:46	hiRoom			D.9		Loud
-SanLorenzoF	47	503	2001:470:f325:503::200:47	hiRoom			D.9		Normal
-TTLB-LBY	48	503	2001:470:f325:503::200:48	hiIDF			M.2		??
-TT2E-2FCat	49	503	2001:470:f325:503::200:49	hiIDF			M.2		??
-TT2D-2FReg	50	503	2001:470:f325:503::200:50	hiIDF			M.2		??
-SpareA		51	503	2001:470:f325:503::200:51	hiIDF			Z.9		??
-SpareB		52	503	2001:470:f325:503::200:52	hiIDF			Z.9		??
-SpareC		53	503	2001:470:f325:503::200:53	hiIDF			Z.9		??
+//Name		Num	MgtVL	IPv6				Type		Hierarchy	Noiselevel	Model
```

Comment:
Its nice to see that youve got the models in here now. Im also glad weve been able to standardize by and large on the ex4200-48p

Reply:
Yeah, all columns after "Type" don't really have any meaning to most of the code (Though Hierarchy is used by the auto pinger script(s) and may get used later by some of the deployment scripts). Adding the model numbers seemed to make sense and standardizing was a goal from very early on in my tenure on the tech team after all the fun we had with no 2 HP switches taking the sam configuration syntax.